### DISCUSSION TITLE
is github.community running on github pages?

### DISCUSSION THREAD

**Discussion by taralshah007** [community/community#22613] [Other Feature Feedback, Questions, & Ideas]
<p>I was wondering if github.community is making use of github pages for its dynamic content?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/taralshah007">@taralshah007</a>,</p>
<p>We are happy you are here! GitHub.Community does not use GitHub pages. Pages aren’t designed for dynamic content as it’s geared towards static site generators and doesn’t support what most would describe as ‘dynamic content.’</p>
<p>Best,</p>
<p>Andrea</p>
